repositories/inmem/inmem.go

```diff
@@ -0,0 +1,66 @@
+package inmem
+
+import (
+	"errors"
+	character "github.com/LuisMG96/academy-go-q42021/repositories/characters"
+	"github.com/gocarina/gocsv"
```

Comment:
For best practices, you can separate the core packages, your packages and third packages like following:

Go core

Project Local 

Third-Party